packages/node-core/src/meta/mmrQuery.service.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import assert from 'assert';
+import {Controller} from '@nestjs/common';
+import {u8aToHex} from '@subql/utils';
+import {MMR} from '@subql/x-merkle-mountain-range';
+import {Sequelize} from '@subql/x-sequelize';
+import {MmrStoreType, NodeConfig} from '../configure/NodeConfig';
+import {MmrProof, MmrPayload} from '../events';
+import {baseMmrService, keccak256Hash, PgBasedMMRDB} from '../indexer';
+import {getExistingProjectSchema} from '../utils';
+
+// Currently only serve MMR query
+export class PgMmrQueryDb extends PgBasedMMRDB {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async setLeafLength(length: number): Promise<number> {
+    throw new Error('setLeafLength in PgMmrQueryDb should never been called');
+  }
+}
+
+@Controller('mmrs')
```

Comment:
Just to be safe
```suggestion
@Controller('mmrQuery')
```

Reply:
I think we want to align with this https://github.com/subquery/subql/issues/913#issuecomment-1114354660 ?